ion/src/simulator/android/build.gradle

```diff
@@ -32,14 +31,17 @@ allprojects {
 apply plugin: 'com.android.application'
 
 android {
-  compileSdkVersion 29
+  namespace = "io.github.upsilon.simulator"
+  compileSdkVersion 34
   defaultConfig {
     applicationId "io.github.upsilon.simulator"
-    minSdkVersion 16
-    targetSdkVersion 29
-    def (major, minor, patch) = System.getenv('OMEGA_VERSION').toLowerCase().tokenize('.').collect{it.toInteger()}
-    versionCode major*1000000 + minor*10000 + patch * 100
-    versionName System.getenv('OMEGA_VERSION')
+    minSdkVersion 21
+    targetSdkVersion 33
+    // def (major, minor, patch) = System.getenv('OMEGA_VERSION').toLowerCase().tokenize('.').collect{it.toInteger()}
+    // versionCode major*1000000 + minor*10000 + patch * 100
+    def date = new Date()
+    versionCode date.format('yyyyMMdd').toInteger()
+    versionName "1.0.1-dev"
```

Comment:
`versionName` should be `System.getenv('UPSILON_VERSION')` I think

Reply:
I don't know why, but when I try to build with what you suggested, the versionCode is set to “null” instead of “1.0.1-dev” (that's why I've typed it manually by the way)…